pkg/plugins/optional/helm/v1alpha/scaffolds/init.go

```diff
@@ -359,6 +359,17 @@ func copyFileWithHelmLogic(srcFile, destFile, subDir, projectName string) error
 		contentStr = injectAnnotations(contentStr, hasWebhookPatch)
 	}
 
+	// Apply NetworkPolicy-specific replacements
```

Comment:
@wjiec 

HI note that the goal of HelmChart is ONLY to package/distribute the solution in this format.
So, how is it done in the default scaffold? We need to ensure that both are doing things the same way, and we cannot address customizations and deviations that are valid only for the HelmCharts. 

In this way, can you please check the default scaffold and let us know:
- Do you think that must be changed? If yes, why and how?
- If not, why would this change only be applied to the HelmCharts?
- Also, what is the problem that this change solves? What is not possible to do with it as it is that will be possible to do after those changes?

Reply:
Hi @camilamacedo86 , I think this feature may not be necessary, but a more precise selector I think would be helpful. In the scenario I'm working in, we need to install multiple controllers into the same namespace for historical reasons, and the original podSelector in NetworkPolicy was too broad and would match controllers we didn't want to open.

With that in mind, I think it's possible to synchronize the default scaffold changes. Since we're currently focused on wanting to use the official solution for helm-based deployments, I'm ignoring the default scaffold, sorry 😅.

Reply:
If you want propose changes then, we need to do that in the default scaffold
Ofen, it would have already end up in the helm charts
We will do that if we still need some week in the Helm.

Reply:
@camilamacedo86 Do you think we should extend this change to the default scaffolding?

Reply:
Yes, I think it's necessary. I'd be more than happy to add this to the default scaffolding :)